### PR TITLE
NMS-15708: various corrections/clarifications in Sentinel docs

### PR DESCRIPTION
**Pull request by dschlenk**
### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://opennms.atlassian.net/) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the Jira link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the Jira issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/NMS-15708

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/flows/sentinel/message-broker/kafka.adoc [hunk 1]
```diff
@@ -25,47 +38,37 @@ ssh -p 8301 admin@localhost
...
 ----
 config:edit org.opennms.core.ipc.sink.kafka.consumer<1>
 config:property-set bootstrap.servers my-kafka-ip-1:9092,my-kafka-ip-2:9092<2>
+config:property-set group.id OpenNMS<3>
 config:update
 ----
 
 <1> Edit the configuration for the flow consumer from Kafka.
 <2> Connect to the following Kafka nodes and adjust the IPs or FQDNs with the Kafka port (9092) accordingly.
+<3> (Optional) Use the same value used for `org.opennms.instance.id` in file `etc/custom.system.properties` as the value of `group.id` when using the same Kafka cluster for multiple {page-component-title} instances.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/flows/sentinel/message-broker/kafka.adoc:58]:
```suggestion
<3> (Optional) When using the same Kafka cluster for multiple {page-component-title} instances, the `group.id` value should be the same as the `org.opennms.instance.id` value in the `etc/custom.system.properties` file.
```

File: docs/modules/operation/pages/deep-dive/flows/sentinel/message-broker/kafka.adoc [hunk 2]
```diff
@@ -25,47 +38,37 @@ ssh -p 8301 admin@localhost
...
+Refer to the configuration used for Kafka by {page-component-title} Core and Minion(s) as the properties required for those components will likely be similar to the properties needed in Sentinel's configuration.
 
-<1> Edit the configuration to send generated events from Sentinel via Kafka.
-<2> Connect to the following Kafka nodes and adjust the IPs or FQDNs with the Kafka port (9092) accordingly.
-
-TIP: To use a Kafka cluster with multiple {page-component-title} instances, customize the topic prefix by setting `group.id`, which is by default set to `OpenNMS`.
-     You can set a different topic prefix for each instance with `config:edit group.id my-group-id` for the consumer and sink.
 
-.Configure the credentials and exit Karaf shell
+.Configure Sentinel to generate and send events (sink) using the same set of properties used in the previous step
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/flows/sentinel/message-broker/kafka.adoc:64]:
```suggestion
.Configure Sentinel to generate and send events (sink) using the same set of properties from the previous step
```

**Review by Bonrob2 — APPROVED** [review]
A few minor updates. Thank you for writing this!

**Review by Bonrob2 — DISMISSED** [review]
LGTM!

File: docs/modules/deployment/pages/sentinel/introduction.adoc [hunk 3]
```diff
@@ -1,17 +1,13 @@
...
-Sentinel provides scalability for data processing, including flows and streaming telemetry.
+Sentinel provides scalability for data processing of flows and streaming telemetry.
 It also supports thresholding for streaming telemetry if you are using OpenNMS xref:deployment:time-series-storage/newts/introduction.adoc#ga-opennms-operation-newts[Newts].
-If you are using Minions or looking for scalable flow processing, you need Sentinel.
+The Sentinel requires a Minion.
+It processes flow and streaming telemetry data received by one or more Minions.
+If you are using Minions to receive flow and/or streaming telemetry and need to scale processing capacity, you need Sentinel.
 
-Sentinel is a Karaf container that handles data processing for OpenNMS and Minion, spawning new containers as necessary to deal with increased data volume.
+As your flow and streaming telemetry volumes increase, additional Sentinel containers can be spawned to meet your processing needs.
```
Comment (mmahacek) [inline docs/modules/deployment/pages/sentinel/introduction.adoc:9]:
```suggestion
As your flow and streaming telemetry volumes increase, additional Sentinel instances can be deployed to meet your processing needs.
```

**Comment by mmahacek** [comment]
This should target `foundation-2023` instead of `release` branch.

**Comment by Bonrob2** [comment]
> This should target `foundation-2023` instead of `release` branch.

I tried rebasing, but the branch was not happy. Suggestions on what to do welcome.

**Comment by mmahacek** [comment]
> I tried rebasing, but the branch was not happy. Suggestions on what to do welcome.

I'm working on this.

**Comment by mmahacek** [comment]
Closing as a new PR #6248 has been created to target the `foundation-2023` branch.

**Pull request closed without merging**